src/main/java/bio/terra/cli/auth/GoogleCredentialUtils.java

```diff
@@ -231,9 +248,9 @@ public static ServiceAccountCredentials getServiceAccountCredential(
   public static AccessToken getAccessToken(GoogleCredentials credential) {
     try {
       credential.refreshIfExpired();
-      return credential.getAccessToken();
     } catch (IOException ioEx) {
-      throw new SystemException("Error refreshing access token", ioEx);
+      logger.warn("Error refreshing access token", ioEx);
```

Comment:
Sometimes the access token will still be valid, in which case we should go ahead and use it without refreshing. If the token is expired, then we'll get permissions errors down the road when we make an authenticated API request.

In normal CLI operation, I expect refreshing the token to always succeed. But when testing, I'm using domain-wide delegation (i.e. Harry Potter users) which doesn't give me a refresh token (at least, I couldn't figure it out). So refreshing always fails, even though the access token is valid because I created it just before the test run.

Reply:
Added a comment.